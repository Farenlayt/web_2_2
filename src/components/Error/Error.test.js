import renderer from "react-test-renderer";
import {Provider} from "react-redux";
import storage from "../../store/FakeStorage";
import Error from "./Error";
import React from "react";

it('Error message is ok.', function() {
    const dom = renderer.create(
        <Provider store={storage}>
            <Error/>
        </Provider>
    ).toJSON();
    expect(dom).toMatchSnapshot();
});


