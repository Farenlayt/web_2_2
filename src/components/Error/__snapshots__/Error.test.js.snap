// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error message in major is ok. 1`] = `
<div>
  <div>
    Ошибочка вышла:
  </div>
  <div>
    Ошибочка вышла
  </div>
  <button
    className="btn"
    onClick={[Function]}
  >
    Удалить
  </button>
</div>
`;

exports[`Error message is ok. 1`] = `
<div>
  <div>
    Ошибочка вышла:
  </div>
  <div>
    Ошибочка вышла
  </div>
  <button
    className="btn"
    onClick={[Function]}
  >
    Удалить
  </button>
</div>
`;
